### PR TITLE
Update SDK version in docs

### PR DESCRIPTION
**Pull request by zsolt-kolbay-sonarsource**

### CODE REVIEW

File: docs/contributing-analyzer.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Visual Studio 2022 version 17.6+ is required to build the project (due to source
     - [.NET Core 2.1 SDK](https://dotnet.microsoft.com/en-us/download/dotnet)
     - .NET Core 3.1 SDK
     - .NET 5.0 SDK
-    - .NET 6.0 SDK
+    - .NET 7.0 SDK
```
Comment (gregory-paidis-sonarsource) [inline docs/contributing-analyzer.md:33]: Should we not include .NET 6  as well?
Is it not needed?
From what I saw it is used at least in `RuleDescriptorGenerator` (which maybe we want to bump, not sure).

**Review by gregory-paidis-sonarsource — CHANGES_REQUESTED** [review]
Left a question

File: docs/contributing-analyzer.md (hunk 1 above)
Comment (zsolt-kolbay-sonarsource) [reply docs/contributing-analyzer.md:34]: `RuleDescriptorGenerator` doesn't seem to cause an issue (at least locally). I removed the .NET 6 SDK and it compiles just fine with the .NET 7 SDK (from VS as well as command line). However, the Net6 project in the ITs fail if the .NET 6 SDK is missing, so I re-added it.

**Review by gregory-paidis-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**